src/nimblepkg/common.nim

```diff
@@ -38,6 +38,7 @@ when not defined(nimscript):
       installExt*: seq[string]
       requires*: seq[PkgTuple]
       bin*: seq[string]
+      binSrc*: seq[string]
```

Comment:
```suggestion
      binSrc*: seq[string]
      binName*: seq[string]
```

Reply:
or maybe even 
```suggestion
      bin*: seq[(string, string)]
```

Reply:
This will cascade into many code changes - everywhere `bin` is referenced.

Reply:
Was able to minimize the changes with a table.